src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -346,4 +346,33 @@ describe('UserManagementComponent', () => {
         comp.selectAllRoles();
         expect(comp.filters.authorityFilter).toEqual(new Set(comp.authorityFilters));
     });
+
+    it('should delete all selected users', () => {
+        const deleteSpy = jest.spyOn(userService, 'deleteUsers').mockReturnValue(of());
+
+        // users
+        const users = [1, 2, 3].map((id) => {
+            const user = new User();
+            user.login = id.toString();
+            return user;
+        });
+
+        comp.selectedUsers = [users[0], users[1]];
+
+        comp.deleteAllSelectedUsers();
+        expect(deleteSpy).toHaveBeenCalledTimes(1);
```

Comment:
I think this is redundant if you check that it's been called with the parameters in the next line.

Reply:
I don't think so that this is redundant, since it can be called multiple times with the same input. So this check is needed :thinking:

Reply:
You're right, there also doesn't seem to be [a function that tests for both](https://stackoverflow.com/questions/40018216/how-to-check-multiple-arguments-on-multiple-calls-for-jest-spies) in one line. Resolved.